diffraction/powder_reduction.py

```diff
@@ -186,10 +182,17 @@ def powder_reduction(sample='sample.nxs',
     # 3. Convert to d-spacing with option of taking calibration into account
     if calibration is None:
         # No calibration data, use standard convert algorithm
-        sample_dspacing = scn.convert(sample_lambda, 'tof', 'dspacing')
+        sample_dspacing = scn.convert(sample_lambda, 'wavelength', 'dspacing', scatter=True)
```

Comment:
Why didn't you use `transform_coords` here?

Reply:
I removed the option to reduce data without calibration. Instead an exception is raised if equal to `None`.

Allowing `calibration=None` would require too many modifications because it is used for the calibration of detectors but also for the definition of groups, which is required to select the relevant groups of sample and vanadium data before normalisation. 

This option will be available in a more complete workflow.

Reply:
I added an intermediate conversion to `tof` before applying `convert_with_calibration`.